database/schema.sql

```diff
@@ -0,0 +1,96 @@
+CREATE TABLE movie (
+    id SERIAL PRIMARY KEY,
+    genre_id INTEGER REFERENCES genre(id),
+    author_id INTEGER REFERENCES author(id),
+    source_id INTEGER REFERENCES source(id),
+    label_id INTEGER REFERENCES label(id),
+    publish_date DATE,
+    archived BOOLEAN
+);
+
+CREATE TABLE source (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+CREATE TABLE authors (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  first_name VARCHAR(30),
+  second_name VARCHAR(30),
+  PRIMARY KEY(id),
+  );
+
+CREATE TABLE games (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  archived BOOLEAN,
+  multiplayer BOOLEAN,
+  last_played_at DATE,
+  label_id INT, 
+  genre_id INT, 
+  source_id INT,
+  author_id INT,
+  PRIMARY KEY(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id),
+  FOREIGN KEY (source_id) REFERENCES sources(id),
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+);
+
+CREATE TABLE genres (
+    id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(150),
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE music_albums (
+    id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE,
+    on_spotify BOOLEAN,
+    archived BOOLEAN,
+    genre_id INT,
+    genre_name VARCHAR(150),
+    author_id INT,
+    author_first_name VARCHAR(150),
+    author_last_name VARCHAR(150),
+    source_id INT,
+    source_name VARCHAR(150),
+    label_id INT,
+    label_name VARCHAR(150),
+    label_color VARCHAR(150),
```

Comment:
- [ ] Take in mind that the `music_albums` table only stores a reference (id) to its associations, and not their entire data, please consider only storing `genre_id`, `author_id`, `source_id`, and `label_id` from them! 🤓 🔧

Reply:
All done, thank you.